### ISSUE TITLE
User cannot access options in Combobox component with keyboard

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1473]
## Environment
Chrome 70
macOS Mojave

## Violation
The user cannot access the drop down box options with a keyboard

**Comment by asudoh** [comment]
Thanks @dakahn for reporting! As we talked offline, combo box is a react-only component as of today, and ones we have in dev env is for demonstrating markup/styles only.
